content/doc/installation/from-binary.en-us.md

```diff
@@ -14,6 +15,21 @@ menu:
 
 # Installation from binary
 
-All downloads come with SQLite, MySQL and PostgreSQL support, and are built with embedded assets. Keep in mind that this can be different for older releases.
+All downloads come with SQLite, MySQL and PostgreSQL support, and are built with embedded assets. Keep in mind that this can be different for older releases. The installation based on our binaries is quite simple, just choose the file matching your platform from the [downloads page](https://dl.gitea.io/gitea), copy the URL and replace the URL within the commands below:
```

Comment:
This is not enabled on the binaries because it increases the file size by something like 15MB? I had a discussion about that with @bkcsoft on Gitter some time ago.